code/modules/atmospherics/machinery/components/trinary_devices/mixer.dm

```diff
@@ -71,39 +71,46 @@
 		return
 
 	//Calculate necessary moles to transfer using PV=nRT
-
 	var/general_transfer = (target_pressure - output_starting_pressure) * air3.volume / R_IDEAL_GAS_EQUATION
 
-	var/transfer_moles1 = node1_concentration * general_transfer / air1.temperature
-
-	var/transfer_moles2 = node2_concentration * general_transfer / air2.temperature
-
-	if((transfer_moles2 <= 0) || (transfer_moles1 <= 0))
-		return
-
+	var/transfer_moles1 = air1.temperature ? node1_concentration * general_transfer / air1.temperature : 0
+	var/transfer_moles2 = air2.temperature ? node2_concentration * general_transfer / air2.temperature : 0
```

Comment:
Use `airx.temperature <= 0` for the conditions

Reply:
<=0 is checked later on in the function, this is just to prevent division by 0